### ISSUE TITLE
ITK's function image_view_from_array produces 4D image with array of >10*9 inputs.

### ISSUE THREAD

**Issue by ViktorvdValk** [InsightSoftwareConsortium/ITKElastix#100]
This is the cause for the memory error in PR #99 

The following script returns an image with DimSize = 10 10 3 2074148458.
While it should return an image with DimSize = 10 10 10 3.

```
def image_generator_3D(x1, x2, y1, y2, z1, z2):
    image = np.zeros([10, 10, 10], np.float32)
    image[z1:z2, y1:y2, x1:x2] = 1
    return image

vector_of_images = np.zeros([3, 10, 10, 10], np.float32)
i = 0
for x in range(0, 5, 2):
    image = image_generator_3D(x, x+5, x, x+5, x, x+5)
    vector_of_images[i] = image
    i += 1

vector_itk = itk.image_view_from_array(vector_of_images.T)
```

Or did I make a mistake here?
Note: Making a 3D image from 3 2D images does work this way.

**Comment by dzenanz** [comment]
If you replace last line by `vector_itk = itk.image_view_from_array(vector_of_images)`, does it work then?

**Comment by ViktorvdValk** [comment]
Noo, same story. The DimSize of the image is then 10 10 3 1660136737. Different but still too big

**Comment by ViktorvdValk** [comment]
When I try to acces the first dimension of the vector_itk object (`vector_itk[0]`) I get this error:

`ValueError: cannot reshape array of size 42220276663395900 into shape (1062156621,3,10,10)`

While the vector_of_images is a regular numpy nd.array of [10,10,10,3], so the itk.image_view_from_array function is somehow increasing the DimSize of the itkImageF4 object.

**Comment by thewtex** [comment]
This will be addressed with this PR: https://github.com/InsightSoftwareConsortium/ITK/pull/2287

**Comment by thewtex** [comment]
Should be addressed with the update to `itk>=5.2rc2`.
